packages/generate-component-docs/src/__snapshots__/contract.test.ts.snap

```diff
@@ -1930,17 +1930,7 @@ exports[`BraidProvider 1`] = `
         | FunctionComponent<LinkComponentProps>
         | { readonly __forwardRef__: ForwardRefExoticComponent<LinkComponentProps & RefAttributes<HTMLAnchorElement>>; }
     styleBody?: boolean
-    theme: {
-        background: { lightMode: string; darkMode: string; }
-        backgroundLightness: { body: "light" | "dark"; bodyDark: "light" | "dark"; brand: "light" | "dark"; brandAccent: "light" | "dark"; brandAccentActive: "light" | "dark"; brandAccentHover: "light" | "dark"; ... 32 more ...; surfaceDark: "light" | "dark"; }
-        color: { foreground: { brandAccentLight: string; brandAccent: string; cautionLight: string; caution: string; criticalLight: string; critical: string; formAccentLight: string; formAccent: string; infoLight: string; ... 14 more ...; secondaryInverted: string; }; background: { ...; }; }
-        displayName: string
-        name: string
-        space: { grid: number; space: { gutter: number; xxsmall: number; xsmall: number; small: number; medium: number; large: number; xlarge: number; xxlarge: number; }; }
-        treatTheme: string
-        vanillaTheme: string
-        webFonts: { linkTag: string; }[]
-    }
+    theme: { vanillaTheme: string; name: string; displayName: string; background: { lightMode: string; darkMode: string; }; webFonts: { linkTag: string; }[]; space: { grid: number; space: { gutter: number; xxsmall: number; ... 5 more ...; xxlarge: number; }; }; color: { ...; }; backgroundLightness: { ...; }; }
```

Comment:
Seems like type normalization is doing something _different_ here for some reason. Nothing really changed, other than the removal of `treatTheme`. Don't think it's anything worth worrying about.